src/OWNERS

```diff
@@ -14,5 +14,9 @@
       pattern: '{cid-*,*-cid-*,*-cid}.js',
       owners: [{name: 'ampproject/wg-analytics'}],
     },
+    {
+      pattern: 'amp-story-embed.js',
+      owners: [{name: 'ampproject/wg-stories'}],
```

Comment:
This group is too wide, can we restrict it to fewer people?
My suggestion:
- enriqe
- newmuis
- gmajoulet

Reply:
Done.